_posts/theme/2020-4-20-uyghur.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: theme
+title: uyghur
+category: theme
+homepage: https://github.com/UyCode/Typora-Uyghur-theme
+download: https://gitee.com/UyCode/images-for-page/raw/master/Typora-Uyghur-Theme.rar
+built-in: false
+author: UyCode
+thumbnail: uyghur.jpg
+typora-root-url: ../../
+typora-copy-images-to: ../../media/theme/onedark
+---
+
+بۇ تېما بولسا Markdownيازىدىغان قۇراللار ئىچىدە ياخشى بولغان قورال Typoraنىڭ ئۇيغۇرچىلاشتۇرۇش ھەققىدىكى تەپسىلاتى.
```

Comment:
Possible to also provide english introduction? 
And is this theme a fork of exiting one?

Reply:
ok, I will provide English version of installation guides. I just copy the yaml part from one-dark for writing md file.